ballerina/Package.md

```diff
@@ -1,14 +1,83 @@
+
+### Creating an App on the DBX Platform
+
+1. Navigate to https://www.dropbox.com/developers/apps and sign up to create an account (or log in if you already have an account)
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-dropbox/main/docs/setup/resources/dropbox-developer-portal.png alt="Dropbox Developer Portal" style="width: 70%;">
+
+2. select “Create app”.
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-dropbox/main/docs/setup/resources/dropbox-apps-dashboard.png alt="Dropbox Apps Dashboard" style="width: 70%;">
+3. Choose an API, type of access you need, and give your app a name.
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-dropbox/main/docs/setup/resources/create-app-page.png alt="Create App Page" style="width: 70%;"> 
+
+For reference, please use the [Dropbox OAuth guide](https://www.dropbox.com/lp/developers/reference/oauth-guide).
+
+### Generating an Access Token
+
+Once you select “Create app”, a page will load that displays information about your newly created app. To generate an access token scroll down to “OAuth 2” and click “Generate” beneath “Generated access token.” The token will display as a long string of characters. Copy this token for use with the Dropbox connector.
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-dropbox/main/docs/setup/resources/app-settings.png alt="App Settings" style="width: 70%;">
+
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the `Dropbox` connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `dropbox` module.
+```ballerina
+import ballerinax/dropbox;
+```
+
+### Step 2: Instantiate a new connector
+
+1. Create a `Config.toml` file and, configure the obtained credentials in the above steps as follows:
+
+```toml
+[authConfig]
+token = "<Enter your token here>"
+```
+
+2. Create a dropbox:ConnectionConfig with the obtained access token and initialize the connector with it.
+
+```ballerina
+configurable http:BearerTokenConfig & readonly authConfig = ?
+dropbox:ConnectionConfig config = {auth: authConfig};
+
+final dropbox:Client dropbox = check new(config, serviceUrl = "https://api.dropboxapi.com/2");
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
+
+**Create a Folder**
+```ballerina
+public function main() returns error? {
+    dropbox:FolderMetadataResponse createFolder = check dropbox->/files/create_folder_v2.post(
+        payload = {
+            path: "/path/to/folder",
+            autorename: true
+        }
+    );
+}
+```
+
+### Step 4: Run the Ballerina application
+
+```ballerina
+bal run
+```
+
 
 ## Examples
```

Comment:
Shall we update this section with the references you have written as examples?

Reply:
I have taken care of examples related documentation in the[ Examples PR](https://github.com/ballerina-platform/module-ballerinax-dropbox/pull/6).